pkg/interface/src/views/apps/chat/components/ChatInput.tsx

```diff
@@ -158,7 +170,7 @@ class ChatInput extends Component<ChatInputProps, ChatInputState> {
         className='cf'
         zIndex={0}
       >
-        <Row p='2' alignItems='center'>
+        <Row p='12px 8px 12px 12px' alignItems='center'>
```

Comment:
isnt this just '3 2 3 3'? we should opt for indigo values before hard coding

Reply:
Unfortunately not -- https://github.com/urbit/indigo-react/blob/1b93c5e502e28fa9e459f5edcb2bd428eeb82585/src/themes/light.ts#L150

Reply:
So our mocks are divergent from Indigo's spacing? Seems weird

Reply:
The mocks are derived from gav's actual indigo chat components, with the exception of the left-hand margin, which is a bit larger for ensuring highlighting will work sanely when that ships, and that timestamps fit in without being too squashed

Reply:
The issue here is that 12px doesn't appear to be an indigo spacing value